Doc/library/math.rst

```diff
@@ -387,8 +387,8 @@ Floating point manipulation functions
 .. function:: issubnormal(x)
 
    Return ``True`` if *x* is a subnormal number, that is a finite
-   nonzero number with a magnitude smaller than the smallest positive normal
-   number, see :data:`sys.float_info.min`.  Return ``False`` otherwise.
+   nonzero number with a magnitude smaller than :data:`sys.float_info.min`.
```

Comment:
The magnitude is an integer, it'll be smaller than `sys.float_info.min` if *x* is smaller than `1`.
```suggestion
   nonzero number smaller than :data:`sys.float_info.min`.
```